actors/albert/main.js

```diff
@@ -92,7 +92,11 @@ function getStartUrl() {
   };
 }
 
-function getCategoryProductsUrl(category, pageNumber = 0, pageSize = 50) {
+function getCategoryProductsUrl(
+  category,
+  pageNumber = 0,
+  pageSize = 20 // previously 50, but Albert client code changed it to 20 so let's do the same
```

Comment:
I experimentally checked how big result we can get, please check it if it really doesn't work for 50.